docs/app/flows/operations.md

```diff
@@ -122,8 +122,9 @@ the original event transaction to the database.
 
 ::: tip Node Modules
 
-For security reasons, module usage is disabled by default. You can configure what Node Modules are available through
-[the `FLOWS_EXEC_ALLOWED_MODULES` environment variable](/self-hosted/config-options#security).
+To prevent unauthorized access to the underlying server, node modules can't be used in the Run Script operation. If you
+require a third party library for your custom script, you can create a custom
```

Comment:
Add ** bold to operation name for consistency

```suggestion
To prevent unauthorized access to the underlying server, node modules can't be used in the **Run Script** operation. If
you require a third party library for your custom script, you can create a custom
```

Reply:
Eagle :eye:!